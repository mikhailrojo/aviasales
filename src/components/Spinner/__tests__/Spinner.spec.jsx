import * as React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import {Spinner} from '../index';

describe('Snapshot', () => {
	it('render Spinner', () => {
		const renderer = new ShallowRenderer();
		const component = renderer.render(<Spinner />);
		expect(component).toMatchSnapshot();
	});
});
