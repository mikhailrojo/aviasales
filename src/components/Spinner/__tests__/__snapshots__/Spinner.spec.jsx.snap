// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot render Spinner 1`] = `
<div
  className="sc-bdVaJa kqwzz"
/>
`;
